.github/workflows/generate-coder-attribution.yml

```diff
@@ -0,0 +1,80 @@
+name: Generate coder attribution
+
+on:
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Does it need to be run periodically? Can't we just run it on release?

Reply:
We can run it on release only. I figure it's cheap to run so doesn't hurt to have a list of authors at a certain point.

Reply:
Thinking more about it - when we improve it to update the latest release draft post, it'll keep the list of authors up to date for us then.